### DISCUSSION TITLE
mysql connection, custom reports

### DISCUSSION THREAD

**Discussion by Rainproof** [frangoteam/FUXA#341] [Ideas]
Hi,
I think it would be nice to add the ability to connect to mysql.
It would also be nice to be able to construct separate reports on user-defined events, not just alarms, in order to place these reports on views.

**Comment by unocelli** [comment]
Hi,
I agree with you, the feature of being able to use an external database to save DAQ, Alarms, Events data seems important to me too

**Reply by GusakIurii** [reply]
can we expect mysql support in the near future?

**Reply by unocelli** [reply]
I added the influxdb connection

**Reply by henjoe** [reply]
@unocelli do you have documentation on how to work on Influxdb connection? Thanks!
